libreyolo/label/page.py

```
@@ -0,0 +1,1077 @@
+  $("#insights").classList.add("show");
+  $("#insbody").innerHTML = `<div class="iload">Analyzing images…</div>`;
+  let d; try{ d = await jget("/api/insights"); }
+  catch(e){ $("#insbody").innerHTML = `<div class="iload">Could not compute insights.</div>`; return; }
+  renderInsights(d);
+}
+function closeInsights(){ $("#insights").classList.remove("show"); }
+function renderInsights(d){
+  const W=d.width, H=d.height, MP=d.megapixels;
+  const maxRes = d.top_resolutions.length ? d.top_resolutions[0][2] : 1;
+  const resBars = d.top_resolutions.map(r=>
+    `<div class="ibar"><span class="il">${r[0]}×${r[1]}</span><span class="it"><span style="width:${Math.round(100*r[2]/maxRes)}%"></span></span><span class="ic">${r[2]}</span></div>`).join("");
```

Ruff (0.15.15)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)